### DISCUSSION TITLE
[FEATURE REQUEST] Overall progress for batch generating

### DISCUSSION THREAD

**Discussion by DJPatron** [hlky/stable-diffusion-webui#215] [Feature suggestions]
**Is your feature request related to a problem? Please describe.**
If I'm using "Batch count (how many batches of images to generate)" slider and there is a bunch of pictures generating, I can see only progress for every picture in the console, but not overall progress.
![image](https://user-images.githubusercontent.com/67385488/187067357-e718a27c-f751-4daa-91bf-14e5512011ef.png)

**Describe the solution you'd like**
Imagine that I want to create 10 images. If first of the images generated succesfully, I want to see something like (in bold text style):

an apocalyptic derelict city street with destroyed buildings on the sides
100%|████████████████████████████████████████████| 128/128 [00:23<00:00,  5.50it/s, **1 of 10**]

**Comment by hlky** [comment]
Should be easy to add, thanks for the suggestion
